doc/tutorials/ann/ann.txt

```diff
@@ -195,31 +195,41 @@ number of features in the thyroid dataset and are just used as an abstract
 representation.
 
 @code
-// Load the training set.
-arma::mat dataset;
-data::Load("thyroid_train.csv", dataset, true);
+#include <mlpack/core.hpp>
+#include <mlpack/methods/ann/layer/layer.hpp>
+#include <mlpack/methods/ann/ffn.hpp>
 
-// Split the labels from the training set.
-arma::mat trainData = dataset.submat(0, 0, dataset.n_rows - 4,
-    dataset.n_cols - 1);
+using namespace mlpack;
+using namespace mlpack::ann;
 
-// Split the data from the training set.
-arma::mat trainLabelsTemp = dataset.submat(dataset.n_rows - 3, 0,
```

Comment:
**Check deleted(red) line number 218 and 207** to understand the first change.
`207:       - arma::mat trainLabelsTemp = dataset.submat(dataset.n_rows - 3, 0, ...`
`218:       - model.Train(trainData, trainLabels);  `
//I should have made 2 commits, that would have made it easy to spot.

Reply:
Most of the time I look at the diff anyway, but this becomes definitely helpful if you take a look at changes at some point in the review process.